documentation/sphinx/source/tls.rst

```diff
@@ -73,22 +72,11 @@ The value for each setting can be specified in more than one way.  The actual va
 
 As with all other command-line options to ``fdbserver``, the TLS settings can be specified in the :ref:`[fdbserver] section of the configuration file <foundationdb-conf-fdbserver>`.
 
-The settings for certificate file, key file, peer verification, password and CA file are interpreted by the loaded plugin.
+The settings for certificate file, key file, peer verification, password and CA file are interpreted by the TLS.
```

Comment:
"interpreted by the TLS" is missing a word, I think.  The TLS implementation?  I'm not sure what wording to suggest here...